tests/multiple_database/tests.py

```diff
@@ -2009,12 +2009,9 @@ def test_pseudo_empty_fixtures(self):
         this shouldn't raise an error (#14068).
         """
         new_io = StringIO()
-        management.call_command("loaddata", "pets", stdout=new_io, stderr=new_io)
-        command_output = new_io.getvalue().strip()
-        # No objects will actually be loaded
-        self.assertEqual(
-            command_output, "Installed 0 object(s) (of 2) from 1 fixture(s)"
-        )
+        warn_msg = "No fixture data found for 'pets'. (File format may be invalid.)"
+        with self.assertWarnsMessage(RuntimeWarning, warn_msg):
+            management.call_command("loaddata", "pets", stdout=new_io, stderr=new_io)
```

Comment:
in [this](https://github.com/django/django/blob/main/django/core/management/commands/loaddata.py#L233C1-L252C44),
`objects` count will be zero because the deserializer method will yield an empty list or generator. Consequently, the `objects_in_fixture` value will consistently be zero, triggering the warning mentioned above.


I'm uncertain about where to handle this effectively. Could you point me toward the right direction for managing it?

Reply:
@felixxm  Do let me know your thoughts on this.